qtism/data/storage/xml/XmlCompactDocument.php

```diff
@@ -55,6 +55,24 @@
  */
 class XmlCompactDocument extends XmlDocument
 {
+    /**
+     * XmlCompactDocument constructor.
+     *
+     * Create a new XmlCompactDocument object.
+     *
+     * @param string $version
+     * @param QtiComponent|null $documentComponent
+     */
+    public function __construct($version = '2.1', QtiComponent $documentComponent = null)
+    {
+        // Version 1.0 was used in legacy code, let's keep it BC.
+        if ($version === '1.0') {
+            $version = '2.1.0';
```

Comment:
![image](https://media3.giphy.com/media/3o7WINomf4UDmyFQXe/giphy.gif)

Reply:
Unfortunately required for legacy.

Reply:
This won't be enough to get everything aligned for legacy and master, but it will be part of another PR.